nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-services-api/pom.xml

```diff
@@ -45,6 +45,11 @@
             <groupId>com.github.stephenc.findbugs</groupId>
             <artifactId>findbugs-annotations</artifactId>
             <version>1.3.9-1</version>
-        </dependency>        
+        </dependency>
```

Comment:
@MikeThomsen Can you please add a `To-do` here explaining that it is an workaround because of a transitive dependency issue. Ideally once the `google-auth-library-oauth2-http` is updated to a version that brings updated jackson artifacts, we can remove this.

Reply:
Done.